### ISSUE TITLE
"Topical listing" is alphabetical, not by topic

### ISSUE THREAD

**Issue by rhiaro** [WebOfTrustInfo/rwot11-the-hague#84]
The "Topical listing" of advance reading papers is not actually sorted by topic.

@elenachachkarova - I can have a go at sorting these unless you prefer to?

**Comment by elenachachkarova** [comment]
@rhiaro, yes thank you, that would be very helpful, I think @ChristopherA said he might have a look as well.
